### ISSUE TITLE
KeyboardListener import Error on new flutter update

### ISSUE THREAD

**Issue by benjamin1571** [f-person/flutter_persistent_keyboard_height#1]
../.pub-cache/hosted/pub.dartlang.org/flutter_persistent_keyboard_height-1.0.0/lib/src/persistent_keyboard_height_provider.dart:54:17: Error: 'KeyboardListener' is imported from both 'package:flutter/src/widgets/keyboard_listener.dart' and 'package:keyboard_utils/keyboard_listener.dart'.
      listener: KeyboardListener(willShowKeyboard: _onWillShowKeyboard),

**Comment by f-person** [comment]
Hi! It seems that you are importing both `keyboard_utils` and ` flutter_persistent_keyboard_height` in a single file. If you remove one of the imports, the error should go away. But if you need to use both packages in a single file you can use an import prefix.

For example, instead of `import 'package:keyboard_utils/keyboard_listener.dart';` you could write
```dart
import 'package:keyboard_utils/keyboard_listener.dart' as keyboard_listener`;
```

and then use the `KeyboardListener` with the `keyboard_listener` import prefix like `keyboard_listener.KeyboardListener`.

**Comment by benjamin1571** [comment]
Okay, Thanks

**Comment by benjamin1571** [comment]
Tried that but still d same error, I used both flutter_persistent_keyboard_height and flutter_keyboard_visibility(it depends on keyboard_utils too)

**Comment by f-person** [comment]
> I used both flutter_persistent_keyboard_height and flutter_keyboard_visibility

Did you try prefixing `flutter_keyboard_visibility` import too?

**Comment by benjamin1571** [comment]
Yes

**Comment by f-person** [comment]
Can you please provide a minimal reproducible example? I tried importing both packages you mentioned (`flutter_keyboard_visibility` and `flutter_persistent_keyboard_height`) and everything works fine for me.

**Comment by f-person** [comment]
Without further information, I'm unable to help you therefore I'm closing the issue. I believe this is not an issue with the package. Please feel free to reopen if this still blocks you and you can provide a minimal reproducible example.

**Comment by fmdogan** [comment]
the package "flutter_persistent_keyboard_height" imports both 'package:flutter/src/widgets/keyboard_listener.dart' and 'package:keyboard_utils/keyboard_listener.dart'. KeyboardListener() is available in both imports.
You, as package creator, should check this.

I edited this in my local. Don't know how to pull pr. This works fine for me.
=> in src>persistent_keyboard_height_provider.dart
line 3-4:
      import 'package:keyboard_utils/keyboard_listener.dart' **_as keyboardListener_**;
      import 'package:keyboard_utils/keyboard_utils.dart';

line 51:
      listener: **_keyboardListener_**.KeyboardListener(willShowKeyboard: _onWillShowKeyboard),

**Comment by f-person** [comment]
Fixed in #2
